lib/Application.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'boxes.dart';
+
+void main() {
+  runApp(const myApp());
+}
+
+class myApp extends StatelessWidget {
+  const myApp({Key? key}) : super(key: key);
```

Comment:
nazywaj klasy z wielkiej litery
```suggestion
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);
```